package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@hyperapp/logger",
-  "version": "0.4.2",
-  "description": "Logger for Hyperapp",
+  "name": "hyperapp-logger",
+  "version": "1.0.1",
+  "description": "Log Hyperapp state updates and action information to the console",
   "main": "dist/logger.js",
   "jsnext:main": "src/index.js",
   "module": "src/index.js",
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/hyperapp/logger.git"
+    "url": "git+https://github.com/hyperappJS/hyperapp-logger.git"
```

Comment:
here and below `hyperappJS` => `hyperapp`?

Reply:
👍

Reply:
That org is so 8 hours ago...